
syntax = "proto3";


import "google/protobuf/empty.proto";


message reqMessage {
    string service_defnition= 1; 
}

message protoParams {}

message priceParamsService {}

message priceRespService {
    float cost_per_process=1;
    string pubk=2;
}

service ProtoDefnition {
    rpc req_msg(protoParams) returns (reqMessage) {}
    rpc req_service_price(priceParamsService) returns (priceRespService) {}

}
